src/js/themes/base.d.ts

```diff
@@ -1395,6 +1395,9 @@ export interface ThemeType {
     searchInput?: ReactComponentElement<any>;
     step?: number;
   };
+  selectMultiple?: {
+    visibleInline?: number;
```

Comment:
Suggest "visibleInline" -> "maxInline" to be a little clearer that this is a maximum. I think "inline" implies "visible".

Reply:
Good idea, I changed this to `maxInline` in the latest commit